### PR TITLE
Correct typing example

### PR DESCRIPTION
**Pull request by Rosi2143**
While trying the example, I found that there are some small errors.

Thanks for the otherwise great docu, though.

### CODE REVIEW

File: _doc/rule.rst [hunk 1]
```diff
@@ -198,27 +198,35 @@ If you want to assign a custom name, you can change the rule name easily by assi
...
     ClassA()
 
 *rule_b.py*::
 
+    import HABApp
+    import typing
+
     if typing.TYPE_CHECKING:            # This is only here to allow
-        from .class_a import ClassA     # type hints for the IDE
+        from .rule_a import ClassA     # type hints for the IDE
```
Comment (spacemanspiff2007) [inline _doc/rule.rst:217]: Could you align it with the comment in the line above?

Comment (Rosi2143) [reply _doc/rule.rst:217]: Sure. I somehow missed this.

**Review by Rosi2143 — COMMENTED** [review]
Comments are now alligned.

**Comment by spacemanspiff2007** [comment]
Perfect - thank you very much!

**Pull request merged**